src/serac/physics/thermal_solid_legacy.hpp

```diff
@@ -5,7 +5,7 @@
 // SPDX-License-Identifier: (BSD-3-Clause)
```

Comment:
This copy right is wrong.. can you grep the code for "Copyright (c) 2019," there seems to be a lot of these slipping through or search/replaces.